WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -42,22 +49,39 @@ struct NotificationsTableViewCellContent: View {
 // MARK: - Regular Style
 fileprivate extension NotificationsTableViewCellContent {
     struct Regular: View {
+
+        @State private var avatarSize: CGSize = .zero
+        @State private var textsSize: CGSize = .zero
+
+        private var rootStackAlignment: VerticalAlignment {
+            return textsSize.height >= avatarSize.height ? .top : .center
+        }
+
         private let info: Style.Regular
 
         fileprivate init(info: Style.Regular) {
             self.info = info
         }
 
         var body: some View {
-            HStack(alignment: .top, spacing: 0) {
+            HStack(alignment: rootStackAlignment, spacing: 0) {
```

Comment:
This is the fix @hassaanelgarem, a dynamic alignment value depending on avatars and texts height